design/ceph/node-loss-rbd-rwo.md

```diff
@@ -0,0 +1,81 @@
+# Recover RBD RWO Pod in case of Node Loss
+
+## Goal
+
+Faster RBD RWO recovery in case of node loss.
+
+## Problem
+
+When a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. If two clients are write to the same volume it could cause corruption. The node must be guaranteed to be down before the volume can be mounted on another node.
+
+## Current Solution
+
+We have a manual solution to the problem which involves forceful deletion of a pod so that forced detachment and attachment work is possible. The problem with the current solution is that even after the forced pod deletion it takes around 11 minutes for the volume to mount on the new node. Also there are still chances of data corruption if the old pod on the lost node comes back online, causing multiple writers and lead to data corruption.
```

Comment:
To be clear, the current solution documents to blocklist the node as seen [here](https://rook.io/docs/rook/latest/Troubleshooting/ceph-csi-common-issues/#node-loss), so the corruption wouldn't happen if those steps are followed.

Reply:
Please clarify in the doc that if the existing steps were followed, corruption was not a risk.

Reply:
in the `current Solution` section right?

Reply:
yes, thanks